frontend/components/ArticleBody.js

```diff
@@ -141,14 +186,132 @@ const bodyStyle = css`
     }
 `;
 
+const profile = colour => css`
+    color: ${colour};
+
+    font-size: 16px;
+    line-height: 20px;
+    font-family: 'Guardian Egyptian Web', Georgia, serif;
+    font-weight: 700;
+    margin-bottom: 4px;
+`;
+
+const shareIcons = css`
+    ${leftCol} {
+        border-bottom: 1px solid ${palette.neutral[5]};
+        padding-bottom: 6px;
+        margin-bottom: 6px;
+    }
+`;
+
+const share = colour => css`
+    border: 1px solid ${palette.neutral[5]};
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    min-width: 32px;
+    max-width: 100%;
+    width: auto;
+    height: 32px;
+    border-radius: 50%;
+    display: inline-block;
+    vertical-align: middle;
+    position: relative;
+
+    fill: ${colour};
+
+    svg {
+        height: 88%;
+        width: 88%;
+        top: 0;
+        bottom: 0;
+        right: 0;
+        left: 0;
+        margin: auto;
+        position: absolute;
+    }
+
+    :hover {
+        background-color: ${colour};
+        border-color: ${colour};
+        fill: white;
+    }
+`;
+
+const ageWarning = colour => css`
+    font-size: 12px;
```

Comment:
Font sizes are becoming a bit haphazard. Maybe we should package these up in pasteup somehow

Reply:
Though I'd probably prefer that to be in a separate PR.